pkg/connector/mapping.go

```diff
@@ -147,6 +149,26 @@ func (tc *TwitterClient) TwitterAttachmentToMatrix(ctx context.Context, portal *
 			return nil, nil, err
 		}
 		attachmentURL = highestBitRateVariant.URL
+	} else if attachment.Card != nil {
+		content := bridgeEvt.MessageEventContent{
+			MsgType:            bridgeEvt.MsgText,
+			BeeperLinkPreviews: []*bridgeEvt.BeeperLinkPreview{tc.attachmentCardToMatrix(ctx, attachment.Card, msg.Entities.URLs)},
+		}
```

Comment:
These should probably include the card/tweet url in body rather than just a link preview